modules/web3/constants/networks.ts

```diff
@@ -56,7 +56,8 @@ export const CHAIN_INFO: ChainInfo = {
     type: 'normal',
     network: SupportedNetworks.MAINNET,
     defaultRpc: NodeProviders.ALCHEMY,
-    spockUrl: process.env.NODE_ENV === 'development' ? STAGING_MAINNET_SPOCK_URL : MAINNET_SPOCK_URL,
+    spockUrl:
+      process.env.NEXT_PUBLIC_VERCEL_ENV === 'development' ? STAGING_MAINNET_SPOCK_URL : MAINNET_SPOCK_URL,
```

Comment:
could the value also be `preview`? Or will it always be `production` or `development`?

Reply:
ah nevermind, just saw your comment in discord